src/assets/labContent.js

```diff
@@ -11,6 +11,7 @@ export default [
   {
     post: {
       profilePic: facebookLogo,
+      index: 0,
```

Comment:
There's no way we can automate this, right? Just double-checking!

Reply:
Actually, I was able to automate this by getting the ProgressBar to find the index of the section using `props.content.indexOf(section)`!